dev/vagrant-config/laravel/dogulp

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
My comment seemed to have disappeared :/

Why are we copying files around to build them? That seems like it'll be introducing a whole bunch of fragility to the process.